front/app/components/SignUpIn/events.ts

```diff
@@ -50,7 +50,7 @@ export function modifyMetaData(
     error: false,
     isInvitation: false,
     token: undefined,
-    inModal: true,
+    inModal: undefined,
```

Comment:
Why undefined instead of true or false?

Reply:
This is for the previously introduced `modifyMetaData` function which had `inModal: true` as a default, but it shouldn't be as it causes issues in the signup page

Reply:
Also I thought undefined could imply it hasn't been set at all. Should I set it to false if that doesnt make sense?

Reply:
I think it's fine to set it to undefined by default, but then we should perhaps enforce defining inModal in `modifyMetaData `?